server/routes/chat_gpt_routes.py

```
@@ -1,20 +1,123 @@
+            for chunk in llm.stream(langchain_messages):
+                chunk_count += 1
+                # chunk는 AIMessageChunk 객체
+                if hasattr(chunk, "content") and chunk.content:
+                    content = chunk.content
+                    # Server-Sent Events 형식으로 데이터 전송
+                    yield f"data: {content}\n\n"
+
+            # 스트리밍 종료 신호
+            yield "data: [DONE]\n\n"
+
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)